test/Dialect/FIRRTL/grand-central.mlir

```diff
@@ -0,0 +1,146 @@
+// RUN: circt-opt -pass-pipeline='firrtl.circuit(sifive-gct)' -split-input-file %s | FileCheck %s
```

Comment:
I think we can drop the `split-input-file`, it gives better error messages that way. I know it's questionable to have multiple circuits per MLIR module, so i'm curious what other people think.

Reply:
My real complaint with _not_ doing `--split-input-file` is that there is then a requirement on name and symbol uniqueness. The `--split-input-file` acts like a namespace so the tests don't get super complicated (and I can reuse the same goofy metasyntactic variables for interface names in these tests... 😬 )

I'm fine to change this, but will let others chime in.

Reply:
The namespaces overlapping seems a bit dangerous. And I think I've seen a bunch of passes which try to dig up exactly one circuit op in the module, and might skip the circuits afterwards entirely.

Reply:
I'll opt to leave this for now.